### DISCUSSION TITLE
Hook name actionMailAlterMessageBeforeSend without the same prefix

### DISCUSSION THREAD

**Discussion by PrestaEdit** [PrestaShop/PrestaShop#36490] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

`actionMailAlterMessageBeforeSend` is the one with `actionMail` suffix while all others are `actionEmail`.

### Expected behavior

Having the same prefix.

### Steps to reproduce

Nothing to reproduce, just read the Mail class

### PrestaShop version(s) where the bug happened

9.0.0-alpha.1

**Comment by Hlavtox** [comment]
That hook is there since 1.7.5 - https://github.com/PrestaShop/PrestaShop/pull/9428

If we change the name (because of one letter):
- We either force module developers to add yet one more version condition into their modules.
- Or add some alias, which have problems on their own.

**Comment by florine2623** [comment]
@PrestaShop/product-council 
Expecting your feedback and decision on that topic !
Thanks!

**Comment by kpodemski** [comment]
It's inconvenient, but the only solution would be adding an alias (as Daniel mentioned, this could lead to some other issues) and not introducing backward incompatible changes.

The alias option is okay, but this is not a bug, this is an improvement that you would like to see. I will convert this into a discussion, and maybe if somebody from the community would be motivated to change it using an alias mechanism, we could take it from there.
